src/lib/ui/ClimateSettingForm/ClimateSettingScheduleFormNameAndSchedule.tsx

```diff
@@ -0,0 +1,111 @@
+import classNames from 'classnames'
+import { type Control, Controller, type FieldValues } from 'react-hook-form'
+
+import { DateTimePicker } from 'lib/ui/DateTimePicker/DateTimePicker.js'
+import { FormField } from 'lib/ui/FormField.js'
+import { InputLabel } from 'lib/ui/InputLabel.js'
+import { TextField } from 'lib/ui/TextField/TextField.js'
+
+import { useDevice } from '../../../hooks.js'
+import { Button } from '../Button.js'
+import { ContentHeader } from '../layout/ContentHeader.js'
+
+interface ClimateSettingScheduleFormNameAndScheduleProps {
+  control: Control<FieldValues, any>
+  deviceId: string
+  onBack: () => void
+  onCancel: (() => void) | undefined
+  onNext: () => void
+  className?: string
```

Comment:
Is this prop needed?

```suggestion
```

Reply:
or if it is required
```suggestion
  className: string | undefined
```